features/largeVideo/components/LargeVideo.js

```diff
@@ -63,9 +64,14 @@ class LargeVideo extends Component {
             this.props.dispatch(participantSelected(activeParticipant.id));
         }
 
+        let mirrored = videoTrack &&
+            videoTrack.isLocal() &&
+            videoTrack.videoType === 'camera';
+
         this.setState({
-            videoStream: videoStream,
-            activeParticipant: activeParticipant
+            activeParticipant: activeParticipant,
+            mirrored: mirrored,
+            videoStream: videoStream
```

Comment:
You used the ES6 object literal property value shorthand above. Please be consistent within your PR!

Generally, I'd like us to use it because of its brevity & I've been modifying non-conforming source code as soon as I see it.